Cargo.toml

```
@@ -8,14 +8,14 @@ edition = "2021"
 codec = { package = "parity-scale-codec", version = "3.0.0" }
 scale-info = { package = "scale-info", version = "2.1.1" }
 clap = { version = "3.2", features = ["derive", "env"] }
-tracing-subscriber = { version = "0.3.14", features = ["env-filter"]   }
+tracing-subscriber = { version = "0.3", features = ["env-filter"]   }
```

we should keep this because https://github.com/tokio-rs/tracing/releases/tag/tracing-subscriber-0.3.13
fixed a compilation error in `version < 0.3.13`

```suggestion
tracing-subscriber = { version = "0.3.14", features = ["env-filter"]   }
```